{{cookiecutter.project_slug}}/Makefile

```diff
@@ -46,56 +46,38 @@ help:
 	@echo "\tsetup-dev-env-full - setup full dev environment to allow IDE completion"
-	@echo "\tsetup-dev-host - setup dev requirements"
+	@echo "\tsetup-dev-host - setup host dev requirements"
 	@echo "\tsetup-pre-commit-hooks - setup pre-commit hooks"
-	@echo "\tsetup-release-tools: - setup tools needed for releasing package"
-{%- if cookiecutter.hosting == "GitLab" %}
-	@echo "\tsetup-cicd-test-stage: - setup tools needed for CI/CD test stage"
-{%- endif %}
-	@echo "\tsetup-cicd-release-stage: - setup tools needed for CI/CD release stage"
 	@echo ""
 	@echo "MAIN TARGETS:"
 	@echo "\tclean - remove all build, test, coverage and Python artifacts"
+	@echo "\ttest - run unit tests"
+	@echo "\tbdd - run bdd tests"
+	@echo "\tlint - check style with flake8"
+	@echo "\tformat - enforce correct format with isort (after a seed-isort-config) and black"
+	@echo "\tformat-check - check format for correctness with isort and black"
+	@echo "\ttype - check Python typing"
 	@echo "\ttox - run tox default targets, usually all tests and checks (see tox.ini)"
 	@echo "\ttox-p - same as 'tox', but with parallel runs"
-	@echo "\trepl - run the repl tool (bpython in our case)"
-	@echo "\tlint - check style with flake8 (uses tox)"
-	@echo "\tformat-check - check format for correctness with isort and black (uses tox)"
-	@echo "\tformat - enforce correct format with isort (after a seed-isort-config) and black (does not use tox)"
-	@echo "\ttype - checks Python typing (uses tox)"
-	@echo "\ttest - run tests quickly with the default Python (3.7 - uses tox)"
-	@echo "\ttest-all - run tests on every Python version declared (uses tox)"
-	@echo "\tbdd - run bdd tests (uses tox)"
-	@echo "\tcoverage - [TO BE IMPLEMENTED] check code coverage quickly with the default Python"
 	@echo "\tdocs - generate Sphinx HTML documentation, including API docs"
 	@echo "\tdocs-pdf - generate Sphinx PDF documentation, including API docs"
-	@echo "\tcicd-release - full release - through ci/cd"
-	@echo "\trelease - [TO BE IMPLEMENTED] full release - from local (requires linux)"
-	@echo "\tbump NEXT_VERSION=[next_version] - bump versions in all relevant files"
-	@echo "\tpublish - publish package to pypi"
-	@echo "\tdist - [TO BE IMPLEMENTED] package"
-	@echo "\tinstall - install the package to the active Python's site-packages"
+	@echo "\trepl - run the repl tool (bpython in our case)"
 	@echo ""
-	@echo "TOX TARGETS:"
-	@echo "\ttox-lint - check style with flake8 (uses tox)"
-	@echo "\ttox-format - check format for correctness with isort and black (uses tox)"
-	@echo "\ttox-type - checks Python typing (uses tox)"
-	@echo "\ttox-test - run tests quickly with the default Python (3.7 - uses tox)"
-	@echo "\ttox-test-all - run tests on every Python version declared (uses tox)"
-	@echo "\ttox-bdd - run bdd tests (uses tox)"
+	@echo "INDIVIDUAL TOX TARGETS:"
+	@echo "\ttox-test - run unit tests with the default Python"
+	@echo "\ttox-test-all - run unit tests on every Python version declared"
```

Comment:
tox-test-all -> tox-test-all-versions
"every Python version**s**" (avec un "s") ou "each Python version". Je pencherais plutôt pour each.

Reply:
used each and done.